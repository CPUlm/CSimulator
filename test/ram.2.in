3
11
1
11
1111
11
1
10
1110
10
0
00
0000
