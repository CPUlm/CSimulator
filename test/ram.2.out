Step 1:
=> o = 0000
Step 2:
=> o = 1111
Step 3:
=> o = 1111
